firestore/integration_test_internal/src/firestore_integration_test.h

```diff
@@ -1,6 +1,7 @@
 #ifndef FIREBASE_FIRESTORE_CLIENT_CPP_SRC_TESTS_FIRESTORE_INTEGRATION_TEST_H_
 #define FIREBASE_FIRESTORE_CLIENT_CPP_SRC_TESTS_FIRESTORE_INTEGRATION_TEST_H_
 
+#include <chrono>  // NOLINT(build/c++11)
```

Comment:
Do we still need these "NOLINT" comments? I thought they were only required in google3?

Reply:
I believe we do (we started doing them way back when everything was still in the iOS repo). I'm not sure if the C++ repo has the same linter checks enabled. We might want to revisit this at some point.